### PR TITLE
Fix license location

### PR DESCRIPTION
**Pull request by fboemer**
* Needed for `he-transformer` to use `ngraph-tf` as external project.

### CODE REVIEW

**Comment by fboemer** [comment]
The build failure seems unrelated:
```
        for v, e in zip(slice_vals, expected):
>           np.testing.assert_array_equal(v, e)
E           AssertionError: 
E           Arrays are not equal
E           
E           (shapes (0, 5), (4, 5) mismatch)
E            x: array([], shape=(0, 5), dtype=float32)
E            y: array([[0.005202, 0.711509, 0.47054 , 0.972025, 0.449617],
E                  [0.029762, 0.312242, 0.009409, 0.92572 , 0.387757],
E                  [0.779166, 0.705877, 0.663553, 0.282641, 0.400346],
E                  [0.223329, 0.150659, 0.951321, 0.948918, 0.264032]], dtype=float32)

test_slice.py:152: AssertionError
=============== 1 failed, 79 passed, 51 skipped in 4.38 seconds ================
```

File: python/CreatePipWhl.cmake [hunk 1]
```diff
@@ -32,58 +32,58 @@ if (PYTHON)
...
+            DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/python/ngraph_bridge/licenses")
+    endforeach()
 
     # Get the list of license files for ngraph
-    file(GLOB NGRAPH_LICENCE_FILES "${NGRAPH_INSTALL_DIR}/licenses/*")
-    
+    file(GLOB NGRAPH_LICENSE_FILES "${NGRAPH_INSTALL_DIR}/licenses/*")
+
     # Copy the licenses for ngraph-tf
-    foreach(DEP_FILE ${NGRAPH_LICENCE_FILES})
```
Comment (fboemer) [inline python/CreatePipWhl.cmake]: The actual fix

File: python/CreatePipWhl.cmake [hunk 2]
```diff
@@ -32,58 +32,58 @@ if (PYTHON)
...
-            DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/python/ngraph_bridge/licenses")        
-    endforeach()            
+        file(COPY ${lic_file_real_path}
+            DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/python/ngraph_bridge/licenses")
+    endforeach()
 
     # Copy the LICENSE at the toplevel
-    file(COPY ${CMAKE_SOURCE_DIR}/../LICENSE 
-        DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/python/ngraph_bridge")        
+    file(COPY ${CMAKE_CURRENT_LIST_DIR}/../LICENSE
```
Comment (fboemer) [inline python/CreatePipWhl.cmake]: The actual fix

**Comment by sayantan-nervana** [comment]
TESTNOW

**Comment by avijit-nervana** [comment]
I fixed this. Please test by either pulling my branch `avijit/fix_compiler_flags_etc` or master when this PR is merged: #346

**Pull request closed without merging**